Add independent disk datasource (#349)
terraform-providers/terraform-provider-vcd: CHANGELOG.md

* Upgrade Terraform SDK dependency to 0.12.8 [#320]
* `resource/vcd_vapp_vm` has new field `computer_name` [#334]
* Import functions can now use custom separators instead of "." [#343]

> * `resource/vcd_independent_disk` Add computed properties (`iops`, `owner_name`, `datastore_name`, `is_attached`, `size_in_bytes`) and full read support [#349]

Reviewer: size_in_bytes shouldn't be here anymore.

Author: updated

> * `resource/vcd_independent_disk` Add computed properties (`iops`, `owner_name`, `datastore_name`, `is_attached`, `size_in_bytes`) and full read support [#349]

Reviewer 2: ```suggestion
* `resource/vcd_independent_disk` Add computed properties (`iops`, `owner_name`, `datastore_name`, `is_attached`) and full read support [#349]
```

* `resource/vcd_independent_disk` Disk state ID changed from name of disk to vCD ID [#349]

> * `resource/vcd_independent_disk` Add new field size_in_bytes [#349]

Reviewer: Leftover.

Author: updated

* Upgrade Terraform SDK dependency to 0.12.8 [#320]
* `resource/vcd_vapp_vm` has new field `computer_name` [#334]
* Import functions can now use custom separators instead of "." [#343]

> * `resource/vcd_independent_disk` Add computed properties (`iops`, `owner_name`, `datastore_name`, `is_attached`) and full read support [#349]

Reviewer: ```suggestion
* `resource/vcd_independent_disk` Add computed properties (`iops`, `owner_name`, `datastore_name`, `is_attached`) and read support for all fields except the `size` [#349]
```

Author: good catch, updated

  * `initscript`
  * `ovf`
  * `accept_all_eulas`

> * `vcd_independent_disk` deprecate field `size`

Reviewer 2: ```suggestion
* ``resource/vcd_independent_disk` deprecate field `size`
```

Author: updated

> * `vcd_independent_disk` deprecate fields `org` and `size`

Reviewer: - `size` we deprecate because we need to switch from MB to Bytes
- But why `org` is being deprecated?

Author: Because we don't use or need it. Disk depends to vdc, not Org. https://code.vmware.com/apis/287/vcloud/doc/doc/operations/POST-CreateDisk.html

Author: org deprecation removed
